universum/modules/output/output.py

```diff
@@ -1,12 +1,15 @@
-from typing import ClassVar, Union
+import sys
+from types import TracebackType
+from typing import ClassVar, Optional, Union
```

Comment:
I believe ClassVar and Union are not used

Reply:
Yes they are. ClassVar is in line 71 and Union is in line 29

Reply:
I see, they were used in non-modified code.